### PR TITLE
Remove mentions of @awaelchli from source code

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Remove mentions of my user name from TODOs. 

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @borda @akihironitta

### CODE REVIEW

**Review by rohitgr7 — APPROVED** [review]
sorry, a little off-topic question. Do you know why we don't call teardown hook of LM and LDM under these exceptions?

File: src/pytorch_lightning/trainer/trainer.py [hunk 1]
```diff
@@ -649,7 +649,6 @@ def _call_and_handle_interrupt(self, trainer_fn: Callable, *args: Any, **kwargs:
                 return self.strategy.launcher.launch(trainer_fn, *args, trainer=self, **kwargs)
             else:
                 return trainer_fn(*args, **kwargs)
-        # TODO(awaelchli): Unify both exceptions below, where `KeyboardError` doesn't re-raise
```
Comment (carmocca) [inline src/pytorch_lightning/trainer/trainer.py:652]: This was about doing the reconcilitation, teardown and state resetting moreso than unifying the excepts

Comment (awaelchli) [reply src/pytorch_lightning/trainer/trainer.py:652]: But will we do it? It's not trivial. I changed my opinion from the past and today prefer not to have my name in todos in the source code. It's a personal preference. 

I can leave the todo and just remove my name.

Comment (carmocca) [reply src/pytorch_lightning/trainer/trainer.py:652]: I'd say yes.

Sure, feel free to use this PR to remove your name. There are several occurrences



```python
src/pytorch_lightning/trainer/connectors/accelerator_connector.py:346:        # TODO: @awaelchli improve the error messages below
src/pytorch_lightning/trainer/connectors/accelerator_connector.py:471:                # TODO: @awaelchli improve error message
src/pytorch_lightning/trainer/connectors/accelerator_connector.py:480:                # TODO: @awaelchli improve error message
src/pytorch_lightning/lite/wrappers.py:102:        # TODO (@awaelchli): let the precision plugin handle the conversion
src/pytorch_lightning/loops/epoch/training_epoch_loop.py:493:        # TODO(@awaelchli): let training/eval loop handle logic around limit_*_batches and val_check_batch
src/pytorch_lightning/loops/fit_loop.py:94:        # TODO(@awaelchli): This setter is required by debugging connector (fast dev run), should be avoided
src/pytorch_lightning/loops/fit_loop.py:105:        # TODO(@awaelchli): This setter is required by debugging connector (fast dev run), should be avoided
src/pytorch_lightning/loops/fit_loop.py:162:        # TODO(@awaelchli): Move track steps inside training loop and move part of these condition inside training loop
src/pytorch_lightning/strategies/strategy.py:240:        # TODO (@awaelchli): standardize this across all plugins in Lightning and Lite. Related refactor: #7324
src/pytorch_lightning/strategies/strategy.py:247:        # TODO (@awaelchli): standardize this across all plugins in Lightning and Lite. Related refactor: #7324
src/pytorch_lightning/strategies/strategy.py:252:        # TODO (@awaelchli): standardize this across all plugins in Lightning and Lite. Related refactor: #7324
```

**Comment by awaelchli** [comment]
> sorry, a little off-topic question. Do you know why we don't call teardown hook of LM and LDM under these exceptions?

Because there are strategies what launch processes with spawn and if you called the hooks on the main process, there wouldn't be anything to teardown. The teardown hooks need to run at the end of each worker process.

**Review by carmocca — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Pull request merged**